react/features/video-quality/middleware.js

```diff
@@ -24,10 +23,6 @@ declare var APP: Object;
  * @returns {Function}
  */
 MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
-    if (action.type === DATA_CHANNEL_OPENED) {
```

Comment:
I'm sure you removed this because we don't need it anymore but I'm having a hard time to figure out why. Could you please explain?